build.assets/windows/build.ps1

```diff
@@ -165,6 +165,27 @@ function Enable-Node {
     }
 }
 
+function Install-Wintun {
+    <#
+    .SYNOPSIS
+        Downloads wintun.dll into the supplied dir
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $InstallDir,
+    )
+    begin {
+        Write-Host "::group::Installing wintun.dll to $InstallDir..."
+        New-Item -Path "$InstallDir" -ItemType Directory -Force | Out-Null
+        $WintunZipfile = "$InstallDir/wintun.zip"
+        Invoke-WebRequest -Uri https://www.wintun.net/builds/wintun-0.14.1.zip -OutFile $WintunZipfile
```

Comment:
For recordkeeping, we had a discussion about direct download in CI vs caching here: https://gravitational.slack.com/archives/C0351AUPJ8K/p1734472487410909?thread_ts=1734465932.932219&cid=C0351AUPJ8K